config.toml

```diff
@@ -1,3 +1,7 @@
-baseURL = 'https://scverse.org'
-languageCode = 'en-us'
-title = 'scverse'
+baseURL = "https://scverse.org"
+languageCode = "en-us"
+title = "scverse"
+description = "Foundational tools for omics data in the life sciences"
+# images = ["https://scverse.org/img/icons/scverse_bw_logo.svg"]
```

Comment:
Why is this commented out?

Reply:
Doesn't seem to work

Reply:
OK. Fine with me if you merge it like this.